packages/modelviewer.dev/data/docs.json

```diff
@@ -5,65 +5,65 @@
       {
         "name": "src",
         "htmlName": "src",
-        "description": "<p>The URL to the 3D model. This parameter is required for <span class=\"attribute\">&lt;model-viewer&gt;</span> to display. Only <a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0\">glTF</a>/<a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#glb-file-format-specification\">GLB</a> models are supported, see <a href=\"https://github.com/google/model-viewer#supported-formats\">Supported Formats</a>.</p>",
+        "description": "The URL to the 3D model. This parameter is required for <span class=\"attribute\">&lt;model-viewer&gt;</span> to display. Only <a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0\">glTF</a>/<a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#glb-file-format-specification\">GLB</a> models are supported, see <a href=\"https://github.com/google/model-viewer#supported-formats\">Supported Formats</a>.",
         "default": [
           "String",
-          "any legal filepath",
-          "empty string"
+          "empty string",
+          "any legal filepath"
         ]
       },
       {
         "name": "alt",
         "htmlName": "alt",
-        "description": "<p>Configures the model with custom text that will be used to describe the model to viewers who use a screen reader or otherwise depend on additional semantic context to understand what they are viewing.</p>",
+        "description": "Configures the model with custom text that will be used to describe the model to viewers who use a screen reader or otherwise depend on additional semantic context to understand what they are viewing.",
         "default": [
           "String",
-          "any description",
-          "empty string"
+          "empty string",
```

Comment:
The type, default, options table seems to have random column widths; can we make those consistent?

Reply:
yep!